File: XV Semana 0ct 2016/alreves/SolutionLittle.py
def es_palindromo(m):
	for i in list(range(len(m))):
		if m[i] != m[len(m)-i-1] :
			return False
	return True
	


n = int(input())
for x in range(n):
	a, b = [int(x) for x in input().split()]
	tot = 0
	for i in list(range(a, b+1)):
		#print(i, end = ' ')
		tot += es_palindromo(str(i))
		#print(tot)
	print(tot)

File: XV Semana 0ct 2016/alreves/SolutionLittle.cpp
#include <iostream>
using namespace std;
int f[1000009];
int es_palindromo(int n)
{
	int c=n, p;
	p=0;
	while (n)
	{
		p *= 10;
		p += n%10;
		n /= 10;
	}
	return p==c;
}

int main()
{
	int a, n, b,tot;
	cin>>n;
	while (n--)
	{
		cin>>a>>b;
		tot =0;
		for (int i = a; i <= b; ++i)
			tot += es_palindromo(i);
		cout<<tot<<endl;
	}
	return 0;
}